nms/scripts/fuji-upgrade/runs-on-nms.sh

```diff
@@ -62,7 +62,7 @@ cat << EOF
               Running yarn script for NMS DB data migration
 --------------------------------------------------------------------------------
 EOF
-pushd /usr/src/node_modules/\@fbcnms/sequelize-models
+pushd /usr/src/node_modules/\@fbcnms/sequelize-models || exit
```

Comment:
With https://github.com/magma/magma/pull/12437/files#diff-58ffecb65a2cd39c9c5318019618d288188704774538df4ce30820f63864bc9aR2, does `fbc` need to be changed here as well?

Reply:
This refers to the archived [fbcnms packages](https://github.com/magma/fbc-js-core), but shouldn't be changed because it runs on an old version.